packages/avatars/stories/examples/Default.tsx

```diff
@@ -0,0 +1,233 @@
+              >
+                <ZendeskIcon role="img" aria-label="Zendesk" />
+              </Avatar>
+            </Col>
+          </Row>
+          <Row alignItems="center" style={{ backgroundColor: surfaceColor, marginBottom: '8px' }}>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+              >
+                <img alt="" src={`images/avatar-3.png`} />
+              </Avatar>
+            </Col>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+                isSystem
+              >
+                <img alt="" src={`images/system.png`} />
+              </Avatar>
+            </Col>
+          </Row>
+          <Row alignItems="center" style={{ backgroundColor: surfaceColor, marginBottom: '8px' }}>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+                backgroundColor={backgroundColor}
+                foregroundColor={foregroundColor}
+              >
+                <Avatar.Text>G</Avatar.Text>
+              </Avatar>
+            </Col>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+                backgroundColor={backgroundColor}
+                foregroundColor={foregroundColor}
+                isSystem
+              >
+                <Avatar.Text>ZD</Avatar.Text>
+              </Avatar>
+            </Col>
+          </Row>
+        </Col>
+      </Row>
+    </Grid>
+  );
+};
+
+Default.argTypes = {
+  size: {
+    name: 'Size',
+    control: {
+      type: 'select',
+      options: ['extraextrasmall', 'extrasmall', 'small', 'medium', 'large']
+    }
+  },
+  status: {
+    name: 'Status',
+    control: {
+      type: 'inline-radio',
+      options: ['none', 'away', 'available']
+    }
+  },
+  badge: {
+    name: 'Badge',
+    control: { type: 'range', min: 0, max: 10, step: 1 }
+  },
+  surfaceColor: {
+    name: 'Surface Color',
+    control: { type: 'color' }
+  },
+  backgroundColor: { name: 'Background Color', control: { type: 'color' } },
+  foregroundColor: {
+    name: 'Foreground Color',
+    control: { type: 'color' }
+  }
+};
+
+Default.args = {
+  size: 'medium',
+  badge: 0,
+  surfaceColor: PALETTE.white,
+  backgroundColor: PALETTE.kale[800],
+  foregroundColor: PALETTE.white
+};
+
+Default.parameters = {
+  docs: {
+    description: {
+      component: `
+  ### Basic
```

Comment:
This heading probably isn't needed. Just have the initial paragraph live under "Avatars".

Reply:
Updated.